README.md

```diff
@@ -7,18 +7,67 @@ Related tools, samples, and specs can be found in the [Azure/iot-plugandplay-mod
 
 ## Submit a model
 
+1. Create a GitHub account if you do not have one yet: [Join GitHub](https://github.com/join).
+
+    [Learn more about GitHub](https://docs.microsoft.com/learn/modules/intro-to-git/)
+
+1. Sign [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure/iot-plugandplay-models?pullRequest=93)
+
 1. Fork the public GitHub repo: [https://github.com/Azure/iot-plugandplay-models](https://github.com/Azure/iot-plugandplay-models).
+
+    [Learn more about forking a repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo)
+
 1. Clone the forked repo. Optionally create a new branch to keep your changes isolated from the `main` branch.
-1. Add the new interfaces to the `dtmi` folder using the folder/filename convention. See the [add-models](#import-a-model-to-the-dtmi-folder) tool below.
+
+    By forking and cloning the public GitHub repo, a copy of repo will be created in your GitHub account and a local copy is created in your dev machine.  Please use this local copy to make modifications.
+
+1. Author a new device model with an unique ID using [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier).  
+
+    Review the [PR requirements](pr-reqs.md) for naming conventions.
+
+    > [!TIP]  
+    > [DTDL Editor for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-dtdl) could help you with the language syntax (including auto-completion) and also validate the syntax with DTDL v2.
+
+1. Save the device model JSON file to a local folder.  
+
+    E.g.
+    `C:\iot-plugandplay-models\MyThermostat.json`
+
 1. Validate the models locally using the `dmr-client` tool to [validate](#validate-models).
+1. Add the new interfaces to the `dtmi` folder using the folder/filename convention. See the [add-models](#import-a-model-to-the-dtmi-folder) tool below.
```

Comment:
I think we should change the language referencing `add-models`, this was carry over from when we were running with js scripts. 

Something like `". See the dmr-client tool import functionality below."`